M2/Macaulay2/packages/Macaulay2Doc/doc3.m2

```diff
@@ -1053,7 +1053,7 @@ document { Key => {(netList, VisibleList),
      Usage => "netList v",
      Inputs => { 
 	  "v" => {"a list of lists of things to be converted to nets and displayed as a table in a net"},
-	  Boxes => Boolean => {"whether to draw boxes around the individual nets"},
+	  Boxes => {"whether to draw boxes around the individual nets. can be a Boolean or a pair of Booleans/lists of rows/columns to insert lines"},
```

Comment:
This description is hard to understand.  Can you rephrase it?  Also capitalize the first word of sentences and end sentences with periods.